docs/develop.md

```diff
@@ -50,3 +50,42 @@ make delete-bootstrap-cluster
 ```
 
 If you have any trouble finding the right command, then you can use `make help` to get a list of all available make targets.
+
+## Toggle between local_mode and remote mode 
+We can retrieve cluster-stacks in two modes. One way is to let the controller fetch it from GitHub which is remote mode and other is we mount the cluster-stacks inside the container and start the controller with appropriate flag. 
+
+> [!NOTE]  
+> Using remote mode is the default behavior. 
+
+Switching between both modes is relatively simple if you're using Tilt. There is a file at the root of the repo `tilt-settings.yaml.example`
+Make a copy of that file with the name of `tilt-settings.yaml`
+```bash
+cp tilt-settings.yaml.example tilt-settings.yaml
+```
+Now, open the file and set the `local_mode` to `true` to use cluster-stacks in local_mode. It should look the following content wise.
+```yaml
+local_mode: true
+```
+
+> [!NOTE]
+> In this mode you need to have cluster-stacks present locally. 
+
+Downloading cluster-stacks can be achieved by many ways but below is a simple way to download it quickly. 
```

Comment:
this won't work as this cluster stack is not adapted for csmctl... Can you adapt the docs so that they contain a local workflow with csmctl?